src/library/parsercsv.cpp

```diff
@@ -48,23 +48,36 @@ QList<QString> ParserCsv::parseAllLocations(const QString& playlistFile) {
                         auto predicate) -> std::optional<std::size_t> {
             const auto it = std::find_if(std::begin(tokens_list), std::end(tokens_list), predicate);
             return (it != std::end(tokens_list))
-                    ? std::distance(std::begin(tokens_list), it)
+                    ? static_cast<std::size_t>(std::distance(std::begin(tokens_list), it))
                     : std::optional<std::size_t>{};
         };
         if (!tokens.isEmpty()) {
             std::optional<std::size_t> locationColumnIndex = detect_location_column(
                     tokens[0],
                     [&](auto i) { return i == QObject::tr("Location"); });
             if ((!locationColumnIndex.has_value()) && tokens.size() > 1) {
-                // Last resort, find column with path separators
-                // This happens in case of csv files in a different language
+                // Last resort, find column with valid path in first row
+                // This happens in case of csv files in a different language,
+                // where the column name is not "Location" If the first row
+                // contains a valid path separator, we assume this is the location column.
+                // - Linux & macOS: Only / is a valid path separator
+                // - Windows: / and \ are valid path separators
+                // This is independent of the existence of the file referred in
+                // the first row, as it's only used for the column detection,
+                // and other rows might contain paths to existing files
                 locationColumnIndex = detect_location_column(tokens[1],
-                        [&](auto i) { return i.contains(QDir::separator()); });
+                        [&](auto i) {
+                            return (i.contains(QDir::separator()) || i.contains(QChar('/')));
```

Comment:
Can we make this code conditional for windows? 

For me is unclear where a file with / is comming form on windows.